website/source/guides/stateful-workloads/portworx.html.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: "guides"
+page_title: "Stateful Workloads with Portworx"
+sidebar_current: "guides-stateful-workloads"
+description: |-
+  There are multiple approaches to deploying stateful applications in Nomad.
+  This guide uses Portworx deploy a MySQL database.
+---
+
+# Stateful Workloads with Portworx
+
+Portworx integrates with Nomad and can manage storage for stateful workloads
+running inside your Nomad cluster. This guide walks you through deploying an HA
+MySQL workload.
+
+## Reference Material
+
+- [Portworx on Nomad][portworx-nomad]
+
+## Estimated Time to Complete
+
+20 minutes
+
+## Challenge
+
+Deploy an HA MySQL database with a replication factor of 3, ensuring the data
+will be replicated on 3 different client nodes.
+
+## Solution
+
+Configure Portworx on each Nomad client node in order to create a storage pool
+that the MySQL task can use for storage and replication.
+
+## Prerequisites
+
+To perform the tasks described in this guide, you need to have a Nomad
+environment with Consul installed. You can use this [repo][repo] to easily
+provision a sandbox environment. This guide will assume a cluster with one
+server node and three client nodes.
+
+~> **Please Note:** This guide is for demo purposes and is only using a single
+server node. In a production cluster, 3 or 5 server nodes are recommended.
+
+## Steps
+
+### Step 1: Ensure Block Device Requirements
+
+* Portworx needs an unformatted and unmounted block device that it can fully
+  manage. If you have provisioned a Nomad cluster in AWS using the environment
+  provided in this guide, you already have an external block device ready to use
+  (`/dev/xvdd`) with a capacity of 50 GB.
+
+* Ensure your root volume's size is at least 20 GB. If you are using the
+  environment provided in this guide, simply add the following line to your
```

Comment:
```suggestion
  environment provided in this guide, add the following line to your
```

nit: Remove simply? - I try to avoid just/simply similar filler words in docs/writing, at least personally?

Reply:
Great feedback. We should indeed avoid using words like that in guides/technical material. Thank you.